python/docs/glossary.md

```diff
@@ -16,38 +16,41 @@ In batch mode, whylogs processes a dataset in batches.
+processed. A **Dataset Profile** is a collection of Column Profiles.
+
+#### Dataset Schema
+A **Dataset Schema** defines the schema for tracking metrics in whylogs. In a dataset schema, the columns of the dataset are mapped to specific data types, and metrics are mapped to each data type. A **Dataset Schema** is created automatically when logging, but it can also be customized by the user, defining column types and/or metrics to be tracked.
 
-#### DatasetProfile
-A **DatasetProfile** is a collection of summary statistics and related metadata for a dataset that whylogs has
-processed.
 
 #### Data Sketching
 **Data sketching** is a class of algorithms that efficiently extract information from large or streaming datasets in a
 single pass. This term is sometimes used to refer specifically to the Apache DataSketches project.
 
-#### Logger
-A **logger** represents the whylogs tracking object for a given dataset (in batch mode) or a collection of data points (
-in streaming mode). A logger is always associated with a timestamp for its creation and a timestamp for the dataset.
-Different loggers may write to different storage systems using different output formats.
-
 #### Metadata
 **Metadata** is data that describes either a dataset or information from whylogs’ processing of the dataset.
 
-#### whylogs file
-[NEED UPDATING] The **whylogs output** is available in the following formats: protobuf, JSON, and flat. Protobuf is a lightweight binary
-format that maps one-to-one with the memory representation of a whylogs object. JSON displays the protobuf data in JSON
-format. Flat outputs multiple files with both CSV and JSON content to represent different views of the data, including
-histograms, upper-bound, lower-bound, and frequent values. To apply advanced transformation on whylogs, we recommend
-using Protobuf.
-
 #### Metric
-TBD New world
+The summary statistics of a dataset is composed by a collection of metrics. Metrics are defined according to its namespaces. Some of the whylogs' default namespaces are: `counts`, `types`, `distribution`, `cardinality`, `frequent items/frequent strings`. By default, a column of a given data type will have a defined set of metrics assigned to it. The metrics to be tracked are customizable according to data type or column name (see [Schema Configuration](examples/basic/Schema_Configuration.ipynb)) . New metrics can also be created in a custom fashion, according to the need of the user (see TBD).
+
 
 ##### Metric component
-NEED TO DEFINE HERE
+A metric component is the smallest unit for a metric.A metric might consist of multiple components. An example is ints metric, which consists of min and max metric components. The calculation of components could be independent or could be coupled with other components.
+
+
+#### whylogs file
+
+A whylogs-generated profile can be stored as a protobuf binary file. Protobuf is a lightweight binary
```

Comment:
I would also state that Protobuf = protocol buffers and that it is a serialization technology, with link to its docs, for users who aren't familiar with it

Reply:
resolved in latest update. thanks!